generator.rb

```diff
@@ -0,0 +1,72 @@
+# establishing the run_generator method
+# getting all of the input needed
+# storing those pieces of input into variables
+# being able to print out those variables
+# iterating through individual characters in words to check which characters are vowels
+
+# wave 1:
+def is_vowel?(vowel)
+    vowels = %w{a e i o u}
+    if vowels.include?(vowel)
+        return true
+    else
+        return false
+    end
+end
+
+# wave 2:
+# reversing word to find first vowel and split it there
+def first_word(word_no_1)
```

Comment:
I like how you broke out finding the words into methods.  Can you see a way to make one method which could serve both words?